source/whats-new.txt

```diff
@@ -40,40 +40,38 @@ Learn what's new in:
 What's New in 5.0
 -----------------
 
-  .. warning:: Breaking Changes in v5.0
-
-    This driver version introduces breaking changes. For a list of these changes, see 
-    the :ref:`Version 5.0 Breaking Changes section <java-breaking-changes-v5.0>` in the 
-    Upgrade guide.
-
-  The 5.0 driver release introduces the following features:
-
-  - Consistent computation of the ``ClusterConnectionMode`` type after
-    instantiating a ``ClusterSettings``. The connection mode defaults to
-    ``SINGLE`` if you specify one host and do not specify a replica set
-    name, or it defaults to ``MULTIPLE`` if you specify more than one
-    host.
-
-  - ``BsonDecimal128`` values respond to method calls in the same way as Java
-    ``Decimal128`` values. In particular, the error responses for the
-    ``isNumber()`` and ``asNumber()`` methods match the Java responses for
-    equivalent ``Decimal128`` values.
-
-  - The ``getElapsedTime()`` method on ``com.mongodb.event.ConnectionReadyEvent``
-    includes the time taken to deliver the ``ConnectionCreatedEvent``. That is,
-    the time returned includes the duration of the
-    ``com.mongodb.event.ConnectionPoolListener.connectionCreated()`` method.
-    
-    The ``getElapsedTime()`` methods on
-    ``com.mongodb.event.ConnectionCheckedOutFailedEvent`` and
-    ``com.mongodb.event.ConnectionCheckedOutEvent`` include the time taken to
-    deliver the ``com.mongodb.event.ConnectionCheckOutStartedEvent``. That is, the
-    time returned includes the duration of the
-    ``com.mongodb.eventConnectionPoolListener.connectionCheckOutStarted()`` method.
-
-  - The ``org.mongodb.scala.Observable.completeWithUnit()`` method is deprecated.
-    This relates to a :ref:`breaking change about Observables in this version
-    <java-breaking-changes-v5.0-observables>`.
+.. warning:: Breaking Changes in v5.0
+
+  This driver version introduces breaking changes. For a list of these changes, see 
+  the :ref:`Version 5.0 Breaking Changes section <java-breaking-changes-v5.0>` in the 
+  Upgrade guide.
+
+The 5.0 driver release introduces the following features:
+
+- ``ClusterSettings`` computes ``ClusterConnectionMode`` more consistently using
```

Comment:
@stIncMale do you think this is a new feature?  I think of it more like like the BsonDecimal128 change below.

Reply:
moved to upgrades